### ISSUE TITLE
Countdown on the nag screen must be removed

### ISSUE THREAD

**Issue by GrayFace** [sandboxie-plus/Sandboxie#4158]
### Is your feature request related to a problem or use case?

The "support Sandboxie" nag screen has a nasty 5 second countdown that has no purpose other than to irritate users.

There is no world in which someone would think "if they're trying to make me hate them with this screen, I should definitely donate money to them".

### Describe the solution you'd like

No countdown

**Comment by DavidXanatos** [comment]
I understand that the countdown screen may be frustrating. However, I would like to emphasize that the screen serves as a gentle reminder of the hard work and dedication that goes into maintaining and improving Sandboxie.

As someone who invests countless hours every day into enhancing the software and providing support, I believe it's fair to ask users to take a moment to consider supporting the project. Your contributions help ensure that Sandboxie continues to evolve and serve its community effectively.

If you find Sandboxie valuable and use it regularly, supporting the project not only removes the reminder screen but also directly contributes to its ongoing development.

Thank you for understanding and for being a part of the Sandboxie community.

**Comment by GrayFace** [comment]
> I understand that the countdown screen may be frustrating. However, I would like to emphasize that the screen serves as a gentle reminder of the hard work and dedication that goes into maintaining and improving Sandboxie.

A gentle reminder would be one without a countdown.

Is there any data that shows that countdown helps sales instead of hindering them?

**Comment by DavidXanatos** [comment]
> 
> Is there any data that shows that countdown helps sales instead of hindering them?

how would you imagine such data to be collected? 

the old sandboxie had such a reminder and a count down so I added one to plus as well.

**Comment by akurpukov** [comment]
For example, unregistered version of Total Commander has similar window every time when you open it, but without countdown, and it is not annoying.

**Comment by MarkSpizu** [comment]
There are many people who do not have a credit card or bank account, so they will never donate. Especially those from the 3rd world. So the nag screen has no purpose. Worst case, once or twice a year would be enough, whoever can/wants to donate will do so.

At the opposite pole there is uBlock Origin where people want to donate but donations are not accepted. 🥹

**Comment by GrayFace** [comment]
> > Is there any data that shows that countdown helps sales instead of hindering them?
> 
> how would you imagine such data to be collected?

For example, by removing the countdown and checking whether the flow of donations goes down or up.

> the old sandboxie had such a reminder and a count down so I added one to plus as well.

Maybe it got it in some later version.


> So the nag screen has no purpose.

It has a purpose, but it should either allow changing its settings somewhere or not actively try to be an annoyance.

> For example, unregistered version of Total Commander has similar window every time when you open it, but without countdown, and it is not annoying.

It only has that "click the right button" thing. WinRAR does it nicely without any shenanigans.

**Comment by CaliSunUSA** [comment]
I don't agree with this request, people just want something for nothing. To stop seeing this countdown, there are two options: Donate or stop using the software. 
Like DavidXanatos said, he spends countless hours on this project, lets give him some respect.

**Comment by GrayFace** [comment]
> I don't agree with this request, people just want something for nothing. To stop seeing this countdown, there are three options: Donate, live with it or stop using the software. Like DavidXanatos said, he spends countless hours on this project, if you can't donate at least lets give him some respect.

You said it yourself. Respect is of importance. Countdown on nag screen is a display of disrespect for the user.

**Comment by CaliSunUSA** [comment]
Respect and gratitude goes to the person that did something not the one that complains. 
So you are saying that DavidXanatos hours of coding and bug fixing is not worth 5 seconds of your time?

**Comment by GrayFace** [comment]
> Respect and gratitude goes to the person that did something not the one that complains.

Respect can't go one way, only both ways.

> So you are saying that DavidXanatos hours of coding and bug fixing is not worth 5 seconds of your time?

Me spending these seconds won't do any good for him.
